pipeline/subject.py

```diff
@@ -0,0 +1,75 @@
+'''
+Schema of subject information.
+'''
+import datajoint as dj
+from . import reference
+
+schema = dj.schema(dj.config.get('database.prefix', '') + 'gi2017_subject')
+
+
+@schema
+class Species(dj.Lookup):
+    definition = """
+    species: varchar(24)
+    """
+    contents = zip(['Mus musculus'])
+
+
+@schema
+class Strain(dj.Lookup): 
+    definition = """ 
+    strain: varchar(24)
+    """
+    contents = zip(['C57BL6','Ai35D','N/A'])
+  
+    
+@schema
+class StrainAlias(dj.Lookup):
+    definition = """  # Other animal strain names that may be used interchangeably in different studies
+    strain_alias: varchar(24)
+    ---
+    -> Strain
+    """
+    contents = [
+            ['C57BL6','C57BL6'],
+            ['B6','C57BL6'],
+            ['Ai35D','Ai35D'],
+            ['N/A','N/A']
+            ]
+
```

Comment:
I'm not sure why this table is necessary. We can just convert them to standard names when ingesting the data.